generate_geofence.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: UTF-8 -*-
+import json
+import urllib2
+import csv
+import configargparse
+
+
+def main():
+
+    # ARGS
+    parser = configargparse.ArgParser(description='geoFenceGen')
+    parser.add_argument(
+        '-dir',
+        '--directory',
```

Comment:
A directory + filename is called a path, or simply file. I would use file or out/outfile as arg name

Reply:
I like that. Can change